tests/testrunner.nim

```diff
@@ -1,4 +1,4 @@
-import os, osproc, re, strutils, terminal, times, pegs
+import os, osproc, strutils, terminal, times, pegs
```

Comment:
can remove the pegs import now

Reply:
pegs are still used: https://github.com/status-im/nim-chronicles/blob/191b09832f426d48ec23603687f9b661ca7b39a8/tests/testrunner.nim#L107